packages/shared/src/components/squads/layout/SquadDirectoryNavbarItem.tsx

```diff
@@ -38,7 +38,11 @@ export function SquadDirectoryNavbarItem({
     >
       <ConditionalWrapper
         condition={!!path}
-        wrapper={(component) => <Link href={path}>{component}</Link>}
+        wrapper={(component) => (
+          <Link href={path} legacyBehavior>
```

Comment:
Probably better to use our internal Link type here: https://github.com/dailydotdev/apps/blob/dc568f15c898c0dec361271109ec0897a5a7c53d/packages/shared/src/components/utilities/Link.tsx

Reply:
Right. But this one is supposed to stay like this because the wrapped component is already an `anchor` tag. So when we start migrating our old Link to the new one, this should still be kept.

Reply:
But it's literally the same no? it's just our wrapper function to always apply legacyBehaviour and make it easier to migrate once we ready?

It doesn't add more then what you have inline here.

Reply:
Agree with Chris here, probably best to keep all links with legacyBehaviour using our component, so when we migrate is easier to differentiate

Reply:
We will not migrate this one since it is intended not to wrap the component with Anchor tag. This is a ConditionalWrapper for a Button component that can be either a `button` an `anchor` tag already. So if we don't add legacyBehavior, we will get the dom nesting issue.

Reply:
If we remove the `legacyBehavior`, since will be wrapped by another anchor tag unnecessarily.

Reply:
We just mean to say use the component I linked above, it's already a wrapper component to achieve the same you doing here.

Reply:
Yes, but the problem is when we migrate everything, that means we will remove `legacyBehavior` on that component, and we should not do it for this case.

Reply:
I get what Lee says, this component will need to handle it by itself no? Maybe we can leave as is but add a TODO or comment here so we know its special.

Reply:
Sure let's add Todo indeed :)